translations/en/LC_MESSAGES/messages.po

```diff
@@ -19,88 +19,183 @@ msgstr ""
 "Generated-By: Babel 2.11.0\n"
 
 msgid "Access Before Assign"
-msgstr "You tried to use the variable {name} on line {access_line_number}, but you set it on line {definition_line_number}. Set a variable before using it."
+msgstr ""
+"You tried to use the variable {name} on line {access_line_number}, but "
+"you set it on line {definition_line_number}. Set a variable before using "
+"it."
+
+msgid "Adventure Progress"
```

Comment:
Messages are added here but do not seem to be added in the general pot file. I suspect that is going to lead to issues in translation. Can you add them to the pot file too? That should just work with the extraction commands explained in contributing.md

Reply:
Fixed this myself!